PWGJE/Tasks/jetSubstructureHF.h

```diff
@@ -27,63 +28,62 @@
 struct JetSubstructureHFTask {
-  Produces<SubstructureTableData> jetSubstructureDataTable;
-  Produces<SubstructureTableMCD> jetSubstructureMCDTable;
-  Produces<SubstructureTableMCP> jetSubstructureMCPTable;
-  Produces<SubstructureTableDataSub> jetSubstructureDataSubTable;
+  o2::framework::Produces<SubstructureTableData> jetSubstructureDataTable;
+  o2::framework::Produces<SubstructureTableMCD> jetSubstructureMCDTable;
+  o2::framework::Produces<SubstructureTableMCP> jetSubstructureMCPTable;
+  o2::framework::Produces<SubstructureTableDataSub> jetSubstructureDataSubTable;
 
-  Produces<SplittingsTableData> jetSplittingsDataTable;
-  Produces<SplittingsTableMCD> jetSplittingsMCDTable;
-  Produces<SplittingsTableMCP> jetSplittingsMCPTable;
-  Produces<SplittingsTableDataSub> jetSplittingsDataSubTable;
+  o2::framework::Produces<SplittingsTableData> jetSplittingsDataTable;
+  o2::framework::Produces<SplittingsTableMCD> jetSplittingsMCDTable;
+  o2::framework::Produces<SplittingsTableMCP> jetSplittingsMCPTable;
+  o2::framework::Produces<SplittingsTableDataSub> jetSplittingsDataSubTable;
 
-  Produces<PairsTableData> jetPairsDataTable;
-  Produces<PairsTableMCD> jetPairsMCDTable;
-  Produces<PairsTableMCP> jetPairsMCPTable;
-  Produces<PairsTableDataSub> jetPairsDataSubTable;
+  o2::framework::Produces<PairsTableData> jetPairsDataTable;
+  o2::framework::Produces<PairsTableMCD> jetPairsMCDTable;
+  o2::framework::Produces<PairsTableMCP> jetPairsMCPTable;
+  o2::framework::Produces<PairsTableDataSub> jetPairsDataSubTable;
 
   // Jet level configurables
-  Configurable<float> zCut{"zCut", 0.1, "soft drop z cut"};
-  Configurable<float> beta{"beta", 0.0, "soft drop beta"};
-  Configurable<float> kappa{"kappa", 1.0, "angularity kappa"};
-  Configurable<float> alpha{"alpha", 1.0, "angularity alpha"};
-  Configurable<bool> doPairBkg{"doPairBkg", true, "save bkg pairs"};
-  Configurable<float> pairConstituentPtMin{"pairConstituentPtMin", 1.0, "pt cut off for constituents going into pairs"};
-  Configurable<std::string> trackSelections{"trackSelections", "globalTracks", "set track selections"};
-  Configurable<bool> applyTrackingEfficiency{"applyTrackingEfficiency", {false}, "configurable to decide whether to apply artificial tracking efficiency (discarding tracks) in jet finding"};
-  Configurable<std::vector<double>> trackingEfficiencyPtBinning{"trackingEfficiencyPtBinning", {0., 10, 999.}, "pt binning of tracking efficiency array if applyTrackingEfficiency is true"};
-  Configurable<std::vector<double>> trackingEfficiency{"trackingEfficiency", {1.0, 1.0}, "tracking efficiency array applied to jet finding if applyTrackingEfficiency is true"};
-
-  Service<o2::framework::O2DatabasePDG> pdg;
+  o2::framework::Configurable<float> zCut{"zCut", 0.1, "soft drop z cut"};
```

Comment:
what is the benefit of not including these common o2 namespaces in the .h files? it seems to me that every single o2 task needs these namespaces so it just seems it makes the headers more complex to read if we remove them